docs/content/latest/quick-start/build-apps/go/ysql-yb-pgx.md

```diff
@@ -0,0 +1,576 @@
+
+- [Go version 1.15](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+You can find the details about the SSL/TLS configuration on the [YSQL PGX](../ysql-pgx/#ssl-tls-configuration) Go driver section. The same configuration is applicable to the YugabyteDB pgx driver as well.
+
+### Go YugabyteDB driver
+
+The [Go YugabyteDB driver package (`pgx`)](https://pkg.go.dev/github.com/yugabyte/pgx) is a fork of jackc/pgx Go driver. The current release of `pgx v4` requires Go modules.
+
+To install the package locally, run the following commands:
+
+```sh
+mkdir yb-pgx
+cd yb-pgx
+go mod init hello
+go get github.com/yugabyte/pgx/v4
+go get github.com/yugabyte/pgx/v4/pgxpool
+```
+
+## Write an application with pgx.Connect()
+
+Create a file `ybsql_hello_world.go` and copy the contents below.
+
+```go
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/yugabyte/pgx/v4"
+)
+
+const (
+	host     = "localhost"
+	port     = 5433
+	user     = "yugabyte"
+	password = "yugabyte"
+	dbname   = "yugabyte"
+	numconns = 12
+)
+
+var connCloseChan chan int = make(chan int)
+var baseUrl string = fmt.Sprintf("postgres://%s:%s@%s:%d/%s",
+	user, password, host, port, dbname)
+
+func main() {
+	// Create a table and insert a row
+	url := fmt.Sprintf("%s?load_balance=true", baseUrl)
+	fmt.Printf("Connection url: %s\n", url)
+	createTable(url)
+	printAZInfo()
+	pause()
+
+	fmt.Println("---- Demonstrating uniform (cluster-aware) load balancing ----")
+	executeQueries(url)
+	fmt.Println("You can verify the connection counts on http://127.0.0.1:13000/rpcz and similar urls for other servers.")
+	pause()
+	closeConns(numconns)
+
+	fmt.Println("---- Demonstrating topology-aware load balancing ----")
+	url = fmt.Sprintf("%s?load_balance=true&topology_keys=cloud1.datacenter1.rack1", baseUrl)
+	fmt.Printf("Connection url: %s\n", url)
+	executeQueries(url)
+	pause()
+	closeConns(numconns)
+
+	fmt.Println("Closing the application ...")
+}
+
+func closeConns(num int) {
+	fmt.Printf("Closing %d connections ...\n", num)
+	for i := 0; i < num; i++ {
+		connCloseChan <- i
+	}
+}
+
+func pause() {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Print("\nPress Enter/return to proceed: ")
+	reader.ReadString('\n')
+}
+
+func createTable(url string) {
+	conn, err := pgx.Connect(context.Background(), url)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
+		os.Exit(1)
+	}
+	defer conn.Close(context.Background())
+
+	var dropStmt = `DROP TABLE IF EXISTS employee`
+	_, err = conn.Exec(context.Background(), dropStmt)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Exec for drop table failed: %v\n", err)
+	}
+
+	var createStmt = `CREATE TABLE employee (id int PRIMARY KEY,
+                                             name varchar,
```

Comment:
See what I mean about the rendering? This is what'll happen in the published docs too :)